addons/html_editor/static/src/utils/dom_info.js

```diff
@@ -731,6 +739,32 @@ export function areSimilarElements(node, node2) {
     );
 }
 
+export function hasSameStyleAttributes(node, node2) {
+    const nodeStyles = node
+        .getAttribute("style")
+        .split(";")
+        .map((style) => style.trim())
+        .filter(Boolean);
+    const node2Styles = node2
```

Comment:
Could you wrap the all `.getAttribute("style").split(";").map((style) => style.trim()).filter(Boolean);` in some local function or maybe a map from an array.

Something like [node, node2].map(...)